datacats/environment.py

```diff
@@ -571,6 +571,17 @@ def fix_storage_permissions(self):
             command='/bin/chown -R www-data: /var/www/storage',
             rw={self.sitedir + '/files': '/var/www/storage'})
 
+    def fix_flickrapi_tokencache_permissions(self):
+        """
+        Set the owner of the flickrapi-tokencache folder to www-data container user
+
+        This folder is required for extensions using the flickrapi,
+        e.g. ckan-galleries
+        """
+        web_command(
+            command='/bin/chown -R www-data: /root/.flickr',
+            rw={self.sitedir + '/flickrapi-tokencache': '/root/.flickr'})
+
```

Comment:
if we set the $HOME setting properly when running the server this and the extra bind mounts won't be necessary